pkg/bootkube/bootstrap.go

```diff
@@ -0,0 +1,84 @@
+package bootkube
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/kubernetes-incubator/bootkube/pkg/asset"
+)
+
+// CreateBootstrapControlPlane seeds static manifests to the kubelet to launch the bootstrap control
+// plane.
+func CreateBootstrapControlPlane(assetDir string, podManifestPath string) error {
+	UserOutput("Running temporary bootstrap control plane...\n")
+
+	// Make secrets temporarily available to bootstrap cluster.
+	if err := os.RemoveAll(asset.BootstrapSecretsDir); err != nil {
+		return err
+	}
+	if err := os.Mkdir(asset.BootstrapSecretsDir, os.FileMode(0755)); err != nil {
+		return err
+	}
+	secretsDir := filepath.Join(assetDir, asset.AssetPathSecrets)
+	secrets, err := ioutil.ReadDir(secretsDir)
+	if err != nil {
+		return err
+	}
+	for _, secret := range secrets {
+		if err := copyFile(filepath.Join(secretsDir, secret.Name()), filepath.Join(asset.BootstrapSecretsDir, secret.Name()), true); err != nil {
```

Comment:
Leaving the secrets as world readable feels a bit wrong - but I'm not immediately sure what expectations we should set of how bootkube is executed and/or if the temp control plane should have a specific user (or privileged). Now that bootkube doesn't have to bind on protected port -- it doesn't really need special permissions, so I think we could scope its access down more.

Maybe we just add a TODO for now to re-asses the temporary secret handling later? I don't think this really should block us right now - as it is only first node on first boot.